qutip/steadystate.py

```diff
@@ -81,7 +81,7 @@ def _default_steadystate_args():
     def_args = {'sparse': True, 'use_rcm': False,
                 'use_wbm': False, 'use_precond': False,
                 'all_states': False, 'M': None, 'x0': None, 'drop_tol': 1e-4,
-                'fill_factor': 100, 'diag_pivot_thresh': None, 'maxiter': 1000,
+                'fill_factor': 100, 'diag_pivot_thresh': 0.1, 'maxiter': 1000,
```

Comment:
I think that the default value of `diag_pivot_thresh` (i.e. the one we get when we pass `None`) is somehow wrong in recent versions of SciPy / SuperLU. The documentation of SuperLU (https://portal.nersc.gov/project/sparse/superlu/ug.pdf) has the default threshold as 1.0 for `splu` but `0.1` for everything else and the `splu` default has a comment that implies it should be `0.1` too.

Setting the default here to `1.0` causes it to fail the splu test, but setting it to anything else causes it to pass.